app/models/project.rb

```diff
@@ -47,8 +47,16 @@ class Project < ActiveRecord::Base
 
   scope :search, ->(name) { where("name like ?", "%#{name}%") }
 
-  def docker_repo(registry)
-    File.join(registry.base, permalink_base)
+  def docker_repo(registry, dockerfile)
+    repo = File.join(registry.base, permalink_base)
+    if suffix = dockerfile.sub(/^Dockerfile\.?/, '').presence
+      repo << "-#{suffix.parameterize}"
+    end
```

Comment:
Maybe add logic for handling the case where `dockerfile` looks like `nginx/Dockerfile` rather the `Dockerfile.nginx`

Reply:
yep, good catch, fixed